### PR TITLE
Added tests for javascript and instructions on how to run them

### PR DESCRIPTION
**Pull request by Jaxobc**

### CODE REVIEW

File: portfolio/src/main/webapp/js/TestingReadMe.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+The tests in custom-script-spec.js require two dependencies I believe
+  The Jasmine framework for the testing.
+    npm install -g jasmine
+  The JSDom library required to utilisie document calls
+    npm install -g jsdom
+
+You need to remove the typewriterFeature() call from the global scope
+to run the tests.
```
Comment (kennzoid) [inline portfolio/src/main/webapp/js/TestingReadMe.txt:9]: Add a TODO here to remove this when the typewriterFeature is removed from global scope.

File: portfolio/src/main/webapp/js/TestingReadMe.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+The tests in custom-script-spec.js require two dependencies I believe
+  The Jasmine framework for the testing.
+    npm install -g jasmine
+  The JSDom library required to utilisie document calls
+    npm install -g jsdom
```
Comment (kennzoid) [inline portfolio/src/main/webapp/js/TestingReadMe.txt:5]: Ideally these would get installed via Makefile. Typically you'd write a "make test" clause and have it depend on these packages.

There's not really enough time to work through that, so just take this as an FYI :p

**Review by kennzoid — APPROVED** [review]

File: portfolio/src/main/webapp/js/custom-script.js [hunk 3]
```diff
@@ -716,3 +715,12 @@ function typewriterFeature() {
     if (MARTYR_NAMES.length) setTimeout(typeName, 2250);
   });
 }
+
+/* eslint-disable */
```
Comment (kennzoid) [inline portfolio/src/main/webapp/js/custom-script.js:693]: Add comment indicating why linting is disabled here.

File: portfolio/src/main/webapp/js/TestingReadMe.txt (hunk 1 above)
Comment (Jaxobc) [reply portfolio/src/main/webapp/js/TestingReadMe.txt:9]: Sure Kenneth

File: portfolio/src/main/webapp/js/TestingReadMe.txt (hunk 2 above)
Comment (Jaxobc) [reply portfolio/src/main/webapp/js/TestingReadMe.txt:5]: Yeah that makes sense

File: portfolio/src/main/webapp/js/custom-script.js (hunk 3 above)
Comment (Jaxobc) [reply portfolio/src/main/webapp/js/custom-script.js:693]: Okay sure.

**Review by kofimeighan — APPROVED** [review]

**Pull request merged**